rules-java-diva/addon/src/main/java/org/jboss/windup/rules/apps/diva/service/DivaStackTraceService.java

```diff
@@ -0,0 +1,109 @@
+package org.jboss.windup.rules.apps.diva.service;
+
+import java.util.List;
+
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.__;
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.model.FileLocationModel;
+import org.jboss.windup.graph.model.FileReferenceModel;
+import org.jboss.windup.graph.model.WindupFrame;
+import org.jboss.windup.graph.model.resource.FileModel;
+import org.jboss.windup.graph.model.resource.SourceFileModel;
+import org.jboss.windup.graph.service.FileLocationService;
+import org.jboss.windup.graph.service.GraphService;
+import org.jboss.windup.reporting.model.InlineHintModel;
+import org.jboss.windup.reporting.model.IssueDisplayMode;
+import org.jboss.windup.rules.apps.diva.model.DivaStackTraceModel;
+import org.jboss.windup.rules.apps.java.model.JavaMethodModel;
+
+public class DivaStackTraceService extends GraphService<DivaStackTraceModel> {
+
+    FileLocationService fileLocationService;
+
+    public DivaStackTraceService(GraphContext context) {
+        super(context, DivaStackTraceModel.class);
+        fileLocationService = new FileLocationService(context);
+    }
+
+    // public static int count0 = 0;
```

Comment:
remove commented out code from this class

Reply:
See 054568a